pcsx2/GameDatabase.cpp

```diff
@@ -761,6 +761,7 @@ const GameDatabaseSchema::GameEntry* GameDatabase::findGame(const std::string_vi
 		return &gameEntry->second;
 	}
 
-	Console.Error(fmt::format("[GameDB] Could not find '{}' in GameDB", serialLower));
+	if (serialLower.size())
```

Comment:
Personally I'd use `!serialLower.empty()` for clarity, but same effect.

Reply:
And will also use an early return, at the top instead, so I don't have to do it for the searching gamedb log too.